cli/src/settings/settings.rs

```diff
@@ -184,10 +184,34 @@ impl Settings {
                 .ok_or_else(|| anyhow!("Could not determine config file name"))?
         };
 
-        let mut document = fs::read_to_string(filename.clone())
-            .context("Could not read existing configuration file")?
-            .parse::<Document>()
-            .context("Could not parse existing configuration file")?;
+        // check if the file exists
+        let exists = match fs::metadata(&filename) {
```

Comment:
I think this could be simplified to [`filename.exists()`](https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.exists), unless there's a benefit to returning the errors at this point? (I suspect that any error that would occur here would be reported by one of the later calls anyway)

Reply:
Oh, thanks!  I was looking for an `exists()` in `std::fs`, not in `Path`.